indra/newview/llfloatereditextdaycycle.cpp

```diff
@@ -433,7 +433,7 @@ void LLFloaterEditExtDayCycle::refresh()
     {
         LLLineEditor* name_field = getChild<LLLineEditor>(TXT_DAY_NAME);
         name_field->setText(mEditDay->getName());
-        name_field->setEnabled(mCanMod);
+        name_field->setEnabled(false);
```

Comment:
#2360 doesn't seem to mention anything about disabling the field. Is this line correct?

Reply:
I think Yes

This name should be modified another waym using Save As or directly in the Library
Moreover, editing this text doesn't take effect because this value is never used later

Reply:
I suggest checking how it works for notecards and scripts and making the same here. Changing this field appears to trigger onCommitName and apply correctly to inventory.

Reply:
> Changing this field appears to trigger onCommitName and apply correctly to inventory.

onCommitName modifies the unpacked copy of the object
The modified name stays unused after applying changes